Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/POSMessage.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.conceptpower.app.core;
+
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Component
+@PropertySource("classpath:pos.properties")
+public class POSMessage {
```

Comment:
@jdamerow I went through the docs of spring:message. We don't have any method for fetching all the keys. Using spring:message we will be able to fetch the values from resource file based on the keys. So should I retrive the keys from Java end by using keySet() in resourcebundle. And iterate those keys in jsp and fetch the data from properties file? Is it advisable?